contracts/contracts/governance/Governable.sol

```diff
@@ -134,6 +134,26 @@ contract Governable {
         }
     }
 
+    /**
+     * @dev Prevents a view contract function from being called
+     * while the contract is inside a reentrant protected method.
+     * This can prevent third party contracts from being attacked
+     * via working with partially updated data.
+     */
+    modifier nonReentrantView() {
```

Comment:
Contracts will be smaller if the modifier calls an internal function. Modifiers are like macros in they copy the code to all functions that are applied to. Copying a call to an internal function is a lot less duplicate code to copying the whole function.

Reply:
This sounds worth doing then. Particularly our main non-reentrant modifier.

Reply:
Sure